src/style.css

```diff
@@ -1,3 +1,62 @@
-h1 {
-  color: blue;
-}
\ No newline at end of file
+
+* {
+  box-sizing: border-box;
+  font-family: NoyhRW01-Medium, sans-serif;
+}
+
+ul {
+  list-style-type: none;
+  background-color: white;
+  box-shadow: 2px 2px 5px 1px rgba(0, 0, 0, 0.2);
+  padding: 10px;
+}
+
+body {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+}
+
+li {
+  border-bottom: 1px solid grey;
+  padding: 10px;
+  display: flex;
+  justify-content: space-between;
+}
+
+li input {
+  border: none;
+  outline: none;
+}
+
+li input:focus {
+  border: none;
+  outline: none;
+}
+
+li input:focus::placeholder {
+  color: transparent;
+}
+
+li a {
+  text-decoration: none;
+  color: inherit;
+  opacity: 0.4;
+}
+
+.clear-item {
+  display: flex;
+  align-items: center;
+  background-color: rgb(248, 246, 246);
+  border-bottom: none;
+  justify-content: center;
+}
+
+i {
+  opacity: 0.4;
+  cursor: move;
+}
+
+section {
+  width: 50%;
+}
```

Comment:
- [ ] Please work on your design and make to make it similar to the project captured in the video below :arrow_down: Note that as a developer it is very important to pay attention to detail. :smile: 

#### Project capture
![68747470733a2f2f696d672e796f75747562652e636f6d2f76692f416355642d5f596a6a71672f302e6a7067 (1)](https://user-images.githubusercontent.com/6615394/193027844-39fe2480-4d27-4b22-ba4c-39f53419fc79.jpeg)

#### Your todo list design
![Screenshot from 2022-10-08 13-37-57](https://user-images.githubusercontent.com/6615394/194710410-f4b96aa8-8ff0-457c-a9a7-c487e6358bc8.png)